.storybook/preview.tsx

```diff
@@ -110,3 +113,38 @@ addParameters({
 });
 
 addons.setConfig(ADDONS_CONFIG);
+
+export const globalTypes = {
```

Comment:
@daigof I removed the `swap theme` button because updating the global styles caused all of the stories are render with additional padding: https://www.chromatic.com/snapshot?appId=5f6e46f1e7081000221523cf&id=5f9857317557760022517a76

Decided to use `@storybook/addon-toolbars` for a more built-in solution.